docs/setup.md

```diff
@@ -51,9 +51,28 @@ brew install gnupg2 git
 * Download and install [GPG4Win](https://www.gpg4win.org/).
 * Download and install [the Windows git installer](https://git-scm.com/download/win).
 
-Alternatively, it can be installed via [chocolatey](https://chocolatey.org/packages/gopass)
+Alternatively, these can be installed via [chocolatey](https://chocolatey.org), 
 
-* `choco install gopass` (requires admin privileges)
+```ps1
+choco install git
+choco install gpg4win
+```
+
+[Scoop](https://scoop.sh), 
+
+```ps1
+scoop install git
+# add Extras bucket
+# scoop bucket add extras
```

Comment:
Is the second one really meant to be commented?
Does it require it or not?

Reply:
I assumed that most scoop users have it activated already. But you're right, it is not really clear. How about

```suggestion
# add Extras bucket, if you haven't already
# scoop bucket add extras
```